projects/js-packages/publicize-components/src/components/media-section/index.js

```diff
@@ -148,36 +144,31 @@ export default function MediaSection() {
 
 	useEffect( () => {
 		try {
-			const setMediaDetails = async () => {
-				const mediaDetails = await getMediaDetails( mediaObject );
+			( async () => {
+				const details = await getMediaDetails( mediaObject );
 
-				if ( mediaDetails.mediaData && mediaDetails.metaData ) {
-					setMediaData( mediaDetails.mediaData );
-					setMetaData( mediaDetails.metaData );
+				if ( details && Object.keys( details ).length ) {
```

Comment:
Does it matter if it's an empty object? I think the only way it could be empty is if we return from the first statement in `getMediaDetails` and we could make that return `null` instead. If the media details weren't empty and now they are, we should probably call `setMediaDetials( {} );`